mysite/ShareBox/views.py

```diff
@@ -1,7 +1,38 @@
 from django.http import HttpResponse
```

Comment:
Order your imports properly. They should be in alphabetical order. Also, `from` imports should come later.